freezeyt/cli.py

```diff
@@ -75,6 +74,21 @@ def main(module_name, dest_path, prefix, extra_pages, config_file, config_var):
             raise click.UsageError('DEST_PATH argument is required')
         config['output'] = {'type': 'dir', 'dir': dest_path}
 
+    if progress is None:
+        if sys.stdout.isatty():
```

Comment:
how can we imagine tty. Once I've got advice to connect mobile to another tty a I had no idea what the guy mean...

Reply:
The abbreviation is originally from ["teletype"](https://en.wikipedia.org/wiki/Teleprinter) but today it usually means "interactive terminal". See [`isatty` docs](https://docs.python.org/3/library/io.html#io.IOBase.isatty).

I have no idea what connecting mobile to another tty should mean.

Reply:
@nappex can you tell us more about the "strange advice"? Maybe we need more context about it ...